src/data-source.ts

```diff
@@ -6,7 +6,7 @@ export const AppDataSource = new DataSource({
   schema: 'public',
   synchronize: false,
   extra: {
-    max: 20,
+    max: 30,
```

Comment:
> If max is 1000 can't we spool even further safely?

@rebelchris its max connections on instance, and since we have multiple pods we can't say 1000 per pods. For example API can scale to max 25, so 30x25 is 750, but we also have bg and other stuff.

Our connections usually spike to 200 in read replica and 300 in master.

Reply:
So this is `30` connections per pool (and each pod has its own pool)